.pipelines/cni/pipeline.yaml

```diff
@@ -107,16 +107,16 @@ stages:
       clusterType: overlay-byocni-nokubeproxy-up
       clusterName: "cilium-overlay"
       nodeCount: ${CILIUM_NODE_COUNT}
-      vmSize: ${CILIUM_VM_SKU}
+      vmSize: ${CILIUM_VM_SIZE}
 
   - template: singletenancy/cniv1-template.yaml
     parameters:
       name: win22_cniv1
       clusterType: windows-cniv1-up
       clusterName: win22-cniv1
       nodeCount: ${WINDOWS_NODE_COUNT}
-      vmSize: ${WINDOWS_LINUX_VM_SKU}
-      windowsVMSize: ${WINDOWS_VM_SKU}
+      vmSize: ${WINDOWS_LINUX_VM_SIZE}
+      vmSizeWin: ${WINDOWS_VM_SIZE}

```

Comment:
```suggestion
      vmSize: ${VM_SIZE}
      vmSizeWin: ${VM_SIZE_WIN}

```

Reply:
For windows clusters we need a 4th var that allows us to control the linux node vmSize independently of the linux clusters, Looking at `VM_SIZE_WIN_LINUX` as a compromise.

Reply:
why can't we use the same VM size as we use in linux clusters?

Reply:
Its not that we can't, but when testing scale with higher node counts having a VM with 16 cores assigned to 2 nodes is overkill. Can't see an instance of the inverse being negative..

Reply:
Agreed upon `VM_SIZE_WINCLUSTER_SYSTEMPOOL` with @rbtr offline to clearly signify the purpose of the env var